
---- EXPECTED OUTPUT
before yield
after yield

Exception: Cannot rewind a generator that was already run in %s:%d
Stack trace:
#0 %s(%d): Generator->rewind()
#1 {main}

Exception: Cannot traverse an already closed generator in %s:%d
Stack trace:
#0 {main}

in generator
---- ACTUAL OUTPUT
Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
before yield
after yield

Exception: Cannot rewind a generator that was already run in /root/php-src/Zend/tests/generators/generator_rewind.php:16
Stack trace:
#0 /root/php-src/Zend/tests/generators/generator_rewind.php(16): Generator->rewind()
#1 {main}

Exception: Cannot traverse an already closed generator in /root/php-src/Zend/tests/generators/generator_rewind.php:30
Stack trace:
#0 {main}

in generator
---- FAILED
